javascript-and-json/main.js

```diff
@@ -0,0 +1,30 @@
+var refBooks = [{
+  title: 'HTLM5 Pocket Reference',
+  author: 'Jennifer Niederst Robbins',
+  isbn: '978-1-449-36335-2'
+},
+{
+  title: 'CSS Pocket Reference',
+  author: 'Eric A. Meyer',
+  isbn: '978-1-449-39903-0'
+},
+{
+  title: 'Javascript Pocket Reference',
+  autho: 'David Flanagan',
+  isbn: '978-1-449-31685-3'
+}];
+
+console.log('refBooks object: ', refBooks);
+console.log('refBooks typeof: ', typeof refBooks);
+
+var JSONrefBooks = JSON.stringify(refBooks);
```

Comment:
you want to avoid capital letters in variable names, that is typically reserved for certain `CONST` variables and `Classes`

Reply:
noted, thank you!